Source/ActivityPub.Client/ActivityPubClient.cs

```diff
@@ -100,6 +103,17 @@ private async Task<ASType> Get(Uri uri, Type targetType, int? maxRecursion, Canc
         return obj;
     }
 
+    private static bool IsContentTypeMatch(MediaTypeHeaderValue actual, ActivityPubOptions.ContentType expected) {
```

Comment:
could we move this logic to the equality method in `ContentType`? https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/records#equality-members